plugins/exporter.koplugin/main.lua

```diff
@@ -40,7 +40,7 @@ local _ = require("gettext")
 
 -- migrate settings from old "evernote.koplugin" or from previous (monolithic) "exporter.koplugin"
 local function migrateSettings()
-    local formats = { "flomo", "html", "joplin", "json", "memos", "my_clippings", "readwise", "text", "xmnote" }
+    local formats = { "flomo", "html", "joplin", "json", "memos", "my_clippings", "nextcloud_notes", "readwise", "text", "xmnote" }
```

Comment:
AFAICT this plugin cannot export to any other Nextcloud facilities other than "Nextcloud Notes". If so, please use the keyword "nextcloud". References to it on the UI are ok as "Nextcloud notes"